File: CetunaProject-SPA/src/app/__models/Alumno.ts
import { Documento } from './Documento';

export interface Alumno {
    id: number;
    cedula: number;
    nombreApellido: string;
    edad: number;
    ciudadActual: string;
    dptoActual: string;
    celular: string;
    lugarTrabajo: string;
    nacionalidad: string;
    fechaModificacion: Date;
    nombres?: string;
    apellidos?: string;
    fecNacimiento?: Date;
    ciudadNacimiento?: string;
    dptoNacimiento?: string;
    direccionParticular?: string;
    lineaBaja?: string;
    direccionLaboral?: string;
    telLaboral?: string;
    fotoUrl?: string;
    documentos?: Documento[];
}

export const EmptyAlumno = (): Alumno => ({
    id: null,
    cedula: null,
    nombreApellido: '',
    edad: null,
    ciudadActual: '',
    dptoActual: '',
    celular: '',
    lugarTrabajo: '',
    nacionalidad: '',
    fechaModificacion: null,
    nombres: '',
    apellidos: '',
    fecNacimiento: null,
    ciudadNacimiento: '',
    dptoNacimiento: '',
    direccionParticular: '',
    lineaBaja: '',
    direccionLaboral: '',
    telLaboral: '',
    fotoUrl: ''
});

File: CetunaProject.API/Models/Alumno.cs
using System;
using System.Collections.Generic;

namespace CetunaProject.API.Models
{
    public class Alumno
    {
        public int Id { get; set; }
        public string Nombres { get; set; }
        public string Apellidos { get; set; }
        public DateTime FecNacimiento { get; set; }
        public string CiudadNacimiento { get; set; }
        public string DptoNacimiento { get; set; }
        public string DireccionParticular { get; set; }
        public string CiudadActual { get; set; }
        public string DptoActual { get; set; }
        public string Celular { get; set; }
        public string LineaBaja { get; set; }
        public string DireccionLaboral { get; set; }
        public string LugarTrabajo { get; set; }
        public string TelLaboral { get; set; }
        public int Cedula { get; set; }
        public string Nacionalidad { get; set; }
        public DateTime FechaModificacion { get; set; }
        public ICollection<DocumentoAlumno> Documentos { get; set; }        
    }
}